packages/extensionmanager/src/companions.tsx

```diff
@@ -139,6 +177,20 @@ export function presentCompanions(
         );
       }
       entries.push(<ul key={'kernel-companion-end'}>{kernelEntries}</ul>);
+      entries.push(
+        <p key="server-companion">
```

Comment:
```suggestion
        <p key={`kernel-companion-${index}`}>
```

Reply:
Thx. I have been abused by the copy/paste.